engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java

```diff
@@ -920,8 +927,8 @@ private void deployNodes(List<OperatorDeployInfo> nodeList) throws IOException
   }
 
   private HashMap<String, ComponentContextPair<Stream, StreamContext>> deployOutputStreams(
-    List<OperatorDeployInfo> nodeList, HashMap<String, ArrayList<String>> groupedInputStreams)
-    throws Exception
+      List<OperatorDeployInfo> nodeList, HashMap<String, ArrayList<String>> groupedInputStreams)
```

Comment:
Isn't the change adding more spaces? continuation indentation looks like more than 2 spaces. I am using apex style for IntelliJ and that has continuation indentation set to 2. 

The problem with apex style (for IntelliJ which is checked in) is that it aligns method parameters which I will fix.